### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by framabot**
Translations update from [Weblate](https://weblate.framasoft.org/projects/pytition/pytitions/) for Pytition/Pytition.

### CODE REVIEW

**Comment by tcitworld** [comment]
@fallen :it:

**Comment by fallen** [comment]
Hey good news Andrea is translating Pytition to Italian!
Shouldn't we wait the translation is 100%-ish before merging it here @tcit ?

**Comment by tcitworld** [comment]
>  Shouldn't we wait the translation is 100%-ish before merging it here @tcitworld ?

As you wish. It wont be used until you activate it anyway. Following changes will be pushed to this PR.

File: pytition/locale/fr_FR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1305,7 +1306,7 @@ msgstr "Voulez-vous vraiment transférer la pétition suivante ?"
 
 #: pytition/petition/templates/petition/transfer_petition.html:54
 msgid "It will be transfered to:"
-msgstr "Elle va être transférée à : "
+msgstr "Elle va être transférée à :"
```
Comment (fallen) [inline pytition/locale/fr_FR/LC_MESSAGES/django.po:1309]: Here I guess the original translation was correct, in French after ':' you need an unbreakable space.

File: pytition/locale/it/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -8,57 +8,62 @@ msgstr ""
...
 
 #: pytition/petition/admin.py:40
 msgid "Number of members"
-msgstr ""
+msgstr "Numero di partecipanti"
 
 #: pytition/petition/admin.py:49
 msgid "Error: {}"
-msgstr ""
+msgstr "Errore"
```
Comment (fallen) [inline pytition/locale/it/LC_MESSAGES/django.po:37]: You need to keep {} for the program to work.
{} will be replaced by an error message.

**Comment by tcitworld** [comment]
Nope, you'll have to add a comment or edit directly on Weblate.

**Comment by fallen** [comment]
I made the changes on weblate, how can I trigger a push to this PR from weblate?

**Comment by tcitworld** [comment]
> I made the changes on weblate, how can I trigger a push to this PR from weblate?

It automatically saves and pushes translations at most every 24 hours. You can also go to https://weblate.framasoft.org/projects/pytition/#repository and save and push manually (you weren't project admin, I just added you).

**Comment by fallen** [comment]
> > I made the changes on weblate, how can I trigger a push to this PR from weblate?
> 
> It automatically saves and pushes translations at most every 24 hours. You can also go to https://weblate.framasoft.org/projects/pytition/#repository and save and push manually (you weren't project admin, I just added you).

Thanks!

**Comment by tcitworld** [comment]
You need to set up the Weblate Github app or create a hook so that Github sends the strings to translate and the updated translations to Weblate as soon as they land in master : https://docs.weblate.org/en/latest/admin/continuous.html#automatically-receiving-changes-from-github

**Comment by fallen** [comment]
@tcitworld thanks for the fixup on weblate!
I've just installed Weblate Github app, there does not seem to be any setting of any sort?
Does it just work out of the box?

**Pull request merged**